x-pack/legacy/plugins/reporting/export_types/png/server/execute_job/index.test.js

```diff
@@ -57,6 +48,12 @@ beforeEach(() => {
 
 afterEach(() => generatePngObservableFactory.mockReset());
 
+const mockElasticsearch = {
+  dataClient: {
+    asScoped: () => ({ callAsCurrentUser: jest.fn() }),
+  },
+};
+
```

Comment:
It's recommended to use the  provided mocks when mocking core services.  (multiple occurences, will comment only once)

```ts
import { elasticsearchServiceMock } from '[..]/src/core/server/mocks';

const mockElasticsearch = elasticsearchServiceMock.createSetupContractMock()
```